torch/csrc/jit/passes/quantization.cpp

```diff
@@ -910,5 +910,48 @@ graph(%self, %scale, %zero_point, %dtype):
   rewriter.RegisterRewritePattern(pattern, replacement);
   rewriter.runOnGraph(graph, filter);
 }
+
+void FoldPrepackedWeightIntoModule(
```

Comment:
Should we recursively run it on submodules?

Reply:
OK